code/obj/item.dm

```diff
@@ -435,7 +435,7 @@
 		SPAWN(1 SECOND)
 			if (!src || !M || !user)
 				return
-			SEND_SIGNAL(M, COMSIG_ITEM_CONSUMED, user, src)
+			SEND_SIGNAL(src, COMSIG_ITEM_CONSUMED, user, src)
```

Comment:
additionally, here, `user` is forcefeeding `M` a thing... and the signal args do not reference `M` at all